pkg/util/system/socket.go

```diff
@@ -8,48 +8,10 @@
 package system
```

Comment:
Why keep this file ? Can't we update the code using `CheckSocketAvailable` to use `pkg/util/system/socket` instead ?